src/Elmah.Io.ElasticSearch/ElasticConnectionConfiguration.cs

```diff
@@ -172,10 +168,10 @@ internal static string GetDefaultIndex(IDictionary config, string connectionStri
         [Obsolete("this is here to support the ES connection string pre 1.2 release")]
         internal static string GetDefaultIndexFromConnectionString(string connectionString)
         {
-            Uri myUri = new Uri(connectionString);
+            var myUri = new Uri(connectionString);

```

Comment:
+1.  We can probably delete this obsolete method now that we're 2 releases beyond 1.2.  But that can be fore another day.

Reply:
Removed!